Content.Server/Botany/Systems/BotanySystem.Produce.cs

```diff
@@ -15,10 +15,13 @@ public void ProduceGrown(EntityUid uid, ProduceComponent produce)
         solutionContainer.RemoveAllSolution();
         foreach (var (chem, quantity) in seed.Chemicals)
         {
-            var amount = FixedPoint2.New(quantity.Min);
-            if (quantity.PotencyDivisor > 0 && seed.Potency > 0)
-                amount += FixedPoint2.New(seed.Potency / quantity.PotencyDivisor);
-            amount = FixedPoint2.New(MathHelper.Clamp(amount.Float(), quantity.Min, quantity.Max));
+            var amount = FixedPoint2.Zero;
+            if (seed.Potency > 0)
```

Comment:
Now the minimum reagent value won't be respected. Notice how in the old code, the minimum is is always added to the product, even if the potency is zero. Is this really intended?

Reply:
I originally had amount start at Min and Lerp from 0 to Max. I did not know potency could be 0; I will fix this, thank you.